ecosystem/indexer-grpc/indexer-grpc-fullnode/src/runtime.rs

```diff
@@ -63,6 +75,12 @@ pub fn bootstrap(
     runtime.spawn(async move {
         let context = Arc::new(Context::new(chain_id, db, mp_sender, node_config));
         let server = FullnodeDataService {
+            context: context.clone(),
+            processor_task_count,
+            processor_batch_size,
+            output_batch_size,
+        };
```

Comment:
can you call this test and switch based on whether the dev flag is turned on?

Reply:
What do you mean by dev flag?

Reply:
added `dev_mode` flag.